### ISSUE TITLE
Add ability to give tables a name

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/onrr.gov-site#2275]
Currently - there's no way to enter a name for a table created within a content block. We could maybe solve this by either allowing for a name to be added to a table, or allowing the CMS editor to add a content tag.

**Comment by R-mohansai** [comment]
Added few properties to check with Tabletool to show headings but it is not giving headings outside the table. So, we have to clone the source code and implement in our code it requires time as well because it is not straight forward.

**Comment by R-mohansai** [comment]
Added few properties to check with Tabletool to show headings but it is not giving headings outside the table. So, we have to clone the source code and implement in our code it requires time as well because it is not straight forward.

**Comment by Maroyafaied** [comment]
@R-mohansai where are the headings showing with new added properties? If the headings are showing as column headings then that feature already existed. Have you cloned the source code? Will this be done at the end of this sprint?

**Comment by R-mohansai** [comment]
@Maroyafaied I have tried adding header tool(plugin) inside Tabletool and access it but that doesn't work not the one already existed if you can see git history you can understand.I will try this feature in this sprint but it might go to next sprint. These CMS tickets are big upgrades we are doing to plugins not like bugs.

**Comment by lpgoldstein** [comment]
#3403 Can be done together

**Comment by cthomasONRR** [comment]
This is now complete for all non-collection tables thanks to the new WYSIWYG editor!
